radix-engine/src/system/bootstrap.rs

```diff
@@ -150,6 +152,7 @@ where
         genesis_data_chunks: Vec<GenesisDataChunk>,
         initial_epoch: u64,
         initial_configuration: EpochManagerInitialConfiguration,
+        initial_time_ms: i64,
```

Comment:
I possibly expected this to be an `Instant` but I see `Instant` is only to second precision. Even though we only store it at minute precision...

In any case, this is fine I guess :D. At some point soon we'll move Clock into EpochManager (ConsensusManager?) and fix the substates.